scriptmodules/emulators/retroarch.sh

```diff
@@ -30,11 +35,15 @@ function sources_retroarch() {
     gitPullOrClone "$md_build/shader" https://github.com/RetroPie/common-shaders.git
     # disable the search dialog
     sed -i 's|menu_input_ctl(MENU_INPUT_CTL_SEARCH_START|//menu_input_ctl(MENU_INPUT_CTL_SEARCH_START|g' menu/menu_entry.c
+    if isPlatform "odroid"; then
+        sed -i 's|struct mali_native_window native_window|fbdev_window native_window|' gfx/drivers_context/mali_fbdev_ctx.c
+    fi
 }
 
 function build_retroarch() {
-    local params=(--disable-x11 --enable-dispmanx --disable-oss --disable-pulse --disable-al --disable-jack --enable-sdl2 --enable-floathard)
-    isPlatform "rpi2" && params+=(--enable-neon)
+    local params=(--disable-x11 --disable-ffmpeg --disable-sdl --disable-oss --disable-pulse --disable-al --disable-jack --enable-floathard)
+    isPlatform "rpi2" && params+=(--enable-dispmanx --enable-sdl2 --enable-neon)
```

Comment:
Shouldn't there be a extra line for rpi1 and rpi2 like `isPlatform "rpi" && params+=(--enable-dispmanx --enable-sdl2)`?